common/reflect_resource_plugin_framework.go

```diff
@@ -24,20 +25,20 @@ func TfSdkToGoSdkStruct(tfsdk interface{}, gosdk interface{}, ctx context.Contex
 	}
 	destVal = destVal.Elem()
 
-	if srcVal.Kind() != reflect.Struct || destVal.Kind() != reflect.Struct {
+	if srcVal.Kind() != reflect.Struct {
```

Comment:
This is for the invalid type right? Do we know where that is coming from?

Reply:
Yeah, sometimes destVal doesn't exist, but maybe I can make this check a bit more robust